dtmi/stmicroelectronics/steval_proteus1/standard_if-1.json

```diff
@@ -0,0 +1,301 @@
+  "@type": "Interface",
+  "description": {
+    "en": "Reports temperature, accelerometer, gyroscope, predictive speed status, predictive acceleration peak status, predictive frequency domain status, battery status"
+  },
+  "displayName": {
+    "en": "Standard interface for PROTEUS"
+  },
+  "contents": [
+    {
+      "@type": "Telemetry",
+      "comment": "Battery Level [%]",
+      "displayName": {
+        "en": "Battery [%]"
+      },
+      "name": "battery",
+      "schema": "double"
+    },
+    {
+      "@type": [ 
+        "Telemetry",
+        "Temperature"
+      ],
+      "comment": "Temperature in degrees Celsius from temperature sensor.",
+      "displayName": {
+        "en": "Temperature"
+      },
+      "name": "temperature",
+      "schema": "double",
+      "unit": "degreeCelsius"
+    },
+    {
+      "@type": "Telemetry",
+      "comment": "Acceleration X/Y/Z components [mg]",
+      "displayName": {
+        "en": "Acc Value [mg]"
+      },
+      "name": "accelerometer",
+      "schema": {
+        "@type": "Object",
+        "description": {
+          "en": "Single acceleration sample. I.e.: {a_x, a_y, a_z}."
+        },
+        "displayName": {
+          "en": "Single acceleration sample."
+        },
+        "fields": [
+          {
+            "name": "a_x",
+            "schema": "double"
+          },
+          {
+            "name": "a_y",
+            "schema": "double"
+          },
+          {
+            "name": "a_z",
+            "schema": "double"
+          }
+        ]
+      }
+    },
+    {
+      "@type": "Telemetry",
+      "comment": "Gyroscope X/Y/Z components [dps]",
+      "displayName": {
+        "en": "Gyro Value [dps]"
+      },
+      "name": "gyroscope",
+      "schema": {
+        "@type": "Object",
+        "description": {
+          "en": "Single angular velocity sample. I.e.: {g_x, g_y, g_z}."
+        },
+        "displayName": {
+          "en": "Single angular velocity sample."
+        },
+        "fields": [
+          {
+            "name": "g_x",
+            "schema": "double"
+          },
+          {
+            "name": "g_y",
+            "schema": "double"
+          },
+          {
+            "name": "g_z",
+            "schema": "double"
+          }
+        ]
+      }
+    },
+    {
+      "@type": "Telemetry",
+      "comment": "Acceleration Peak Status X/Y/Z components [m/sˆ2]",
+      "displayName": {
+        "en": "Acc Peak Status [m/sˆ2]"
+      },
+      "name": "predictiveaccelerationstatus",
+      "schema": {
+        "@type": "Object",
+        "description": {
+          "en": "Sample format I.e.: {statusacc_x, statusacc_y, statusacc_z, accpeak_x, accpeak_y, accpeak_z}."
+        },
+        "displayName": {
+          "en": "Single Predictive Acceleration Peak Status sample."
+        },
+        "fields": [
+          {
+            "displayName": {
+              "en": "[AccPeak] Status Acceleration X:"
```

Comment:
Remove ':' from end of displayName value, here and below

Reply:
done